modules/improvedigitalBidAdapter.js

```diff
@@ -1,176 +1,150 @@
-    } else {
-      if (publisherId) {
-        normalizedBidRequest.publisherId = publisherId;
-      }
-      if (placementKey) {
-        normalizedBidRequest.placementKey = placementKey;
+    return requestObj.requests;
+  },
+
+  /**
+   * Unpack the response from the server into a list of bids.
+   *
+   * @param {*} serverResponse A successful response from the server.
+   * @return {Bid[]} An array of bids which were nested inside the server.
+   */
+  interpretResponse: function (serverResponse, request) {
+    const bids = [];
+    utils._each(serverResponse.bid, function (bidObject) {
+      if (!bidObject.price || bidObject.price === null ||
+        bidObject.hasOwnProperty('errorCode') ||
+        typeof bidObject.adm !== 'string') {
+        return;
       }
-    }
 
-    if (keyValues) {
-      normalizedBidRequest.keyValues = keyValues;
-    }
-    if (localSize && localSize.w && localSize.h) {
-      normalizedBidRequest.size = {};
-      normalizedBidRequest.size.h = localSize.h;
-      normalizedBidRequest.size.w = localSize.w;
-    }
-    if (bidId) {
-      normalizedBidRequest.id = bidId;
-    }
-    if (adUnitId) {
-      normalizedBidRequest.adUnitId = adUnitId;
-    }
-    return normalizedBidRequest;
-  }
-
-  let submitNoBidResponse = function(bidRequest) {
-    let bid = bidfactory.createBid(CONSTANTS.STATUS.NO_BID, bidRequest);
-    bid.bidderCode = IMPROVE_DIGITAL_BIDDER_CODE;
-    bidmanager.addBidResponse(bidRequest.placementCode, bid);
-  };
-
-  $$PREBID_GLOBAL$$.improveDigitalResponse = function(response) {
-    let bidRequests = utils.getBidderRequestAllAdUnits(IMPROVE_DIGITAL_BIDDER_CODE);
-    if (bidRequests && bidRequests.bids && bidRequests.bids.length > 0) {
-      utils._each(bidRequests.bids, function (bidRequest) {
-        let bidObjects = response.bid || [];
-        utils._each(bidObjects, function (bidObject) {
-          if (bidObject.id === bidRequest.bidId) {
-            if (!bidObject.price || bidObject.price === null) {
-              submitNoBidResponse(bidRequest);
-              return;
-            }
-            if (bidObject.errorCode && bidObject.errorCode !== 0) {
-              submitNoBidResponse(bidRequest);
-              return;
-            }
-            if (!bidObject.adm || bidObject.adm === null || typeof bidObject.adm !== 'string') {
-              submitNoBidResponse(bidRequest);
-              return;
-            }
-
-            let bid = bidfactory.createBid(CONSTANTS.STATUS.GOOD, bidRequest);
-
-            let syncString = '';
-            let syncArray = (bidObject.sync && bidObject.sync.length > 0) ? bidObject.sync : [];
+      let bid = {};
+      let nurl = '';
+      if (bidObject.nurl && bidObject.nurl.length > 0) {
+        nurl = `<img src="${bidObject.nurl}" width="0" height="0" style="display:none">`;
+      }
+      bid.ad = `${nurl}<script>${bidObject.adm}</script>`;
+      bid.adId = bidObject.id;
+      bid.bidderCode = spec.code;
```

Comment:
Our docs are being updated now, but... you don't need to set this property. In fact if you do, it'll break adapter aliasing.